File: Arrays/Easy/secondLargest.js
const sLargest=(arr) =>{
    let largest=-1;
    let sLargest=-1;
    let n=arr.length;
    for(let i=0;i<n;i++){
        if(arr[i]>largest){
            sLargest=largest;
            largest=arr[i];
        }else if(sLargest<arr[i] && arr[i]!=largest){
            sLargest=arr[i];
        }
    }
    return sLargest;

}
let a=[2,4,6,7,10,9];
console.log(sLargest(a));

File: Arrays/Easy/secondLargest.java
import java.util.*;
public class secondLargest {
    public static void main(String[] args) {
        int[] a={2,4,5,9,8,7,5};
        int ans=secondLargest(a);
        System.out.println(ans);

        
    }
    public static int secondLargest(int[] arr){
        int sLargest=-1;
        int Largest=-1;

        int n=arr.length;
        for(int i=0;i<n;i++){
            if(arr[i]>Largest){
                sLargest=Largest;
                Largest=arr[i];
            }else if(sLargest<arr[i] && arr[i]!=Largest){
                sLargest=arr[i];
            }
        }
        return sLargest;

    }
}
